device/iot-device-samples/multiplexing-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/MultiplexingSample.java

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package samples.com.microsoft.azure.sdk.iot;
+
+import com.microsoft.azure.sdk.iot.device.*;
+import com.microsoft.azure.sdk.iot.device.transport.IotHubConnectionStatus;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+/**
+ * Sample that demonstrates creating a multiplexed connection to IoT Hub using AMQPS / AMQPS_WS. It also demonstrates
+ * removing and adding device clients from the multiplexed connection while it is open.
+ */
+public class MultiplexingSample
+{
+    // Every multiplexed device will maintain its own connection status callback. Because of that, you can monitor
+    // if a particular device session goes offline unexpectedly. This connection status callback is also how you
+    // confirm when a device client is connected after registering it to an active multiplexed connection since the .registerDeviceClient(...)
+    // call behaves asynchronously when the multiplexing client is already open. Similarly, this callback is used to track
+    // when a device client is closed when unregistering it from an active connection.
+    public static class MultiplexedDeviceConnectionStatusChangeTracker implements IotHubConnectionStatusChangeCallback
+    {
+        public boolean isOpen = false;
```

Comment:
So anyone can change this, not just this class?

Reply:
I can make these fields private and make getters so they are readonly, sure